### DISCUSSION TITLE
Chia's blockchain_v2_mainnet.sqlite is so big that it keeps growing until the computer can't fit it?

### DISCUSSION THREAD

**Discussion by joesdu** [Chia-Network/chia-blockchain#17078] [Support]
Chia's blockchain_v2_mainnet.sqlite is so big that it keeps growing until the computer can't fit it?

**Comment by BrandtH22** [comment]
@joesdu , unfortunately the nature of blockchains is that node have to store a copy of the database for the network to be auditable and secure. 

There has been discussion of building a light version of the full node that would not need to store the entire database but that has not been developed nor is there an etc for it (but anyone in the community can build this also).

The current v2 database should be around 140GB which is more than 2.5 years worth of the blockchain database and the size is growing rather deterministically so one can plan that in another 2.5 years the size of the database will be at least 280 GB (more usage over the next two years compared to the previous will increase it even more).

It is good to plan for the database to keep growing and to ensure it is on a fast ssd to ensure there are no issues with keeping up with the chain. 
We do have these instructions to help with migrating the database to another drive and are available to assist in our discord server (https://discord.gg/chia):
https://docs.chia.net/faq/#how-do-i-move-the-chia-blockchain-database-to-another-drive
